COORDINATION.md

```diff
@@ -6,9 +6,81 @@ Reference [BOOTSEQ.md](./BOOTSEQ.md)
+| 14 – Keyboard Controller Initialization                                    | The system firmware or BIOS is initializing the keyboard controller.                      |         |
+| 15 – Embedded Controller/Management Controller Initialization              | The system firmware or BIOS is initializing the embedded management controller.           |         |
+| 16 – Docking Station Attachment                                            | The main system unit is attaching to the docking station.                                 |         |
+| 17 – Enabling Docking Station                                              | The system firmware or BIOS is enabling the docking station.                              |         |
+| 18 – Docking Station Ejection                                              | The main system unit is ejected from the docking station.                                 |         |
+| 19 – Disabling Docking Station                                             | The system firmware or BIOS is disabling the docking station.                             |         |
+| 20 – Calling Operating System Wake-Up Vector                               | The system firmware or BIOS is starting the operating system.                             |         |
+| 21 – Starting Operating System Boot Process (for example, calling INT 19h) | The system firmware or BIOS is booting the operating system.                              |         |
+| 22 – Baseboard or Motherboard Initialization                               | The BIOS is initializing the motherboard.                                                 |         |
+| 23 – Floppy Initialization                                                 | The BIOS is initializing the floppy drive.                                                |         |
+| 24 – Keyboard Test                                                         | The BIOS is testing the keyboard.                                                         |         |
+| 25 – Pointing Device Test                                                  | The BIOS is testing the pointing device.                                                  |         |
+| 26 – Primary Processor Initialization                                      | The BIOS is initializing the primary processor.                                           |         |
+
+
+
+Reference:
+- https://www.dmtf.org/dsp/DSP0249 DMTF DSP0249 State Set Specification
+- https://www.dmtf.org/dsp/DSP0248 DMTF DSP0248 PLDM for Platform Monitoring and Control Specification
+
+### PLDM RDE
+Reference:
+- https://www.dmtf.org/dsp/DSP0218 DMTF DSP0218 PLDM for Redfish Device Enablement
+
+### i2c
+I2C on it's own is not useful unless there is a protocol defined.  
+
+### NC-SI OEM
+
+### usb
+Could use PLDM State Sensor over PLDM over MTCP over USB
+Reference:
+- https://www.dmtf.org/sites/default/files/standards/documents/DSP0283_0.1.5WIP10.pdf (WIP) DMTF MCTP over USB Binding Specification
```

Comment:
let's also add an option for ETH over USB... pros/cons... trust establishment... higher level FW vs SW and so on....